### PR TITLE
Fix #2569. Get flash rtflame example working.

### LINKED ISSUE #2569: flash variable plot params.xmin missing
```txt
Jun 06 01:06:27     6     0 /home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/job_driver/__init__.py:274:_receive DockerDriver(a=NeL8 k=sequential u=PNS1 [_Op(analysis, WsQc)]) error msg={'agentId': 'NeL8hCyIlVfXZmD8ekb0Q8b90HRdFLK9', 'opId': 'WsQcVk4yxil23RVHMBgFpxL3Vqh3IvM1', 'opName': 'analysis', 'reply': {'error': 'report not generated', 'stack': 'Traceback (most recent call last):
File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/pkcli/job_cmd.py", line 202, in _do_get_simulation_frame
msg.data.copy().pkupdate(run_dir=msg.runDir),
File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/template/template_common.py", line 350, in sim_frame_dispatch
res = o(frame_args)
File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/template/flash.py", line 329, in sim_frame_varAnimation
xdomain = [params['xmin'], params['xmax']]
KeyError: 'xmin'
Exception was printed at:
KeyError: 'xmin'
File "/home/vagrant/.pyenv/versions/py3/bin/sirepo", line 8, in <module>
sys.exit(main())
File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/sirepo_console.py", line 18, in main
return pkcli.main('sirepo')
File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/pykern/pkcli/__init__.py", line 108, in main
res = argh.dispatch(parser, argv=argv)
File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/argh/dispatching.py", line 174, in dispatch
for line in lines:
File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/argh/dispatching.py", line 277, in _execute_command
for line in result:
File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/argh/dispatching.py", line 260, in _call
result = function(*positional, **keywords)
File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/pkcli/job_cmd.py", line 50, in default_command
sirepo.template.import_module(msg.simulationType)
File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/pkcli/job_cmd.py", line 181, in _do_fastcgi
sirepo.template.import_module(m.simulationType)
', 'state': 'error'}}
```

### PR DESCRIPTION
**Pull request by e-carlin**
Looks like py2 vs py3 differences.

### CODE REVIEW

File: sirepo/pkcli/admin.py [hunk 1]
```diff
@@ -104,9 +104,9 @@ def setup_dev_proprietary_code(sim_type, rpm_url):
     assert pkconfig.channel_in('dev'), \
         'Only to be used in dev. channel={}'.format(pkconfig.cfg.channel)
 
+    s = sim_data.get_class(sim_type)
```
Comment (robnagler) [inline sirepo/pkcli/admin.py:107]: remove this change, because it conflicts the #2586

File: sirepo/template/flash.py [hunk 2]
```diff
@@ -480,7 +481,7 @@ def _parameters(f):
     res = {}
     for name in ('integer scalars', 'integer runtime parameters', 'real scalars', 'real runtime parameters'):
         for v in f[name]:
-            res[v[0].strip()] = v[1]
+            res[pkcompat.from_bytes(v[0].strip())] = v[1]
```
Comment (robnagler) [inline sirepo/template/flash.py:484]: Other codes don't need to do this with h5py. Why is this happening here?

Comment (e-carlin) [reply sirepo/template/flash.py:484]: The honest truth is I don't really know. Below is what I do know.

The value `v` is a compound datatype (`ascii str`, `int32`). We have to parse the ascii str ourselves. The ascii str then becomes the key for the dictionary. In other cases that I tested (ex warpvnd) it seems we only use path names as the keys for the dictionary we never use data memebers so we never have to do any parsing ourselves. Should I do more digging or is that good?

File: sirepo/pkcli/admin.py (hunk 1 above)
Comment (e-carlin) [reply sirepo/pkcli/admin.py]: https://github.com/radiasoft/sirepo/pull/2580/commits/d605d20b82b846260157ed0ad2bf276556bc4b0a

**Pull request merged**